TTHAnalysis/python/tools/leptonJetReCleaner.py

```diff
@@ -212,26 +264,31 @@ def __call__(self,event):
         ret['minMllAFSS'] = minMllTL(lepsl, lepsl, paircut = lambda l1,l2 : l1.charge ==  l2.charge) 
         ret['minMllSFOS'] = minMllTL(lepsl, lepsl, paircut = lambda l1,l2 : l1.pdgId  == -l2.pdgId) 
 
+        loosetaus=[]; rettlabel = {}; tauret = {}; 
+        loosetaus = self.recleanTaus(tausc, tausd, lepsl if self.cleanTausWithLoose else lepsc, self.label, rettlabel, tauret)
```

Comment:
recleanTaus requires 8 arguments. need to add "event" as additional argument to this call